internal/envoy/v3/ratelimit.go

```diff
@@ -163,3 +163,12 @@ func enableXRateLimitHeaders(enable bool) ratelimit_filter_v3.RateLimit_XRateLim
 	}
 	return ratelimit_filter_v3.RateLimit_OFF
 }
+
+// rateLimitPerRoute returns a per-route config to configure vhost rate limits.
+func rateLimitPerRoute(mode int) *anypb.Any {
```

Comment:
nit we should probably cast to int here and change the function signature to take a `dagRoute.RateLimitPerRoute.VhRateLimits`